### PR TITLE
Support Xenial, Java 8 by default, and Java 8 on Trusty

### PR DESCRIPTION
**Pull request by johnsca**

### CODE REVIEW

File: reactive/openjdk.sh [hunk 1]
```diff
@@ -32,6 +32,17 @@ function install() {
     local java_type=$(config-get 'java-type')
     local java_major=$(config-get 'java-major')
 
+    if grep -q trusty /etc/lsb-release; then
+        # ensure that Java 8 can be installed on trusty
+        add-apt-repository -y ppa:openjdk-r/ppa
+        apt-get update -qq
+    fi
+
+    if [[ `uname -p` == 'ppc64'* ]]; then
```
Comment (kwmonroe) [inline reactive/openjdk.sh:41]: This looks weird as all get out to me, and I'm not near a prompt on a ppc box to verify. To earn the prestigious +1 you so desire, prove to me that the weird 'ppc64'\* string does what you want it to do, that is, match ppc64le.

Comment (johnsca) [inline reactive/openjdk.sh:41]: It's identical to [java-installer.sh](https://bazaar.launchpad.net/~bigdata-dev/bigdata-data/trunk/view/head:/common/noarch/java-installer.sh#L40) but if you want further proof:

``` bash
$ test_cpu_arch="ppc64le"
$ if [[ "$test_cpu_arch" == 'ppc64'* ]]; then
>   echo "Power match"
> fi
Power match
```

Edit: Documented in [the note under ==](http://www.tldp.org/LDP/abs/html/comparison-ops.html#SCOMPARISON2).

**Comment by kwmonroe** [comment]
Thx for the proof.  LGTM.

**Comment by kwmonroe** [comment]
promulgated as cs:openjdk-1

**Pull request merged**